CONTRIBUTING.md

```diff
@@ -0,0 +1,128 @@
+For example, if I want to work on a `feature` called `login-form`, the branch name will be **feature/login-form**. If it is going to be a `fix` in `form-data`, the branch name will be `fix/form-data`.
+Command example:
+
+```
+git checkout -b feature/login-form
+```
+
+6. **Just do it!**
+
+Perform the tasks you wanted to.
+
+7. **Committing your changes**
+
+Now you have made the changes, though they are saved in your system, Git doesn't know what changes you've done. So you have to **commit** your changes. First step is to add the files which you want to add to the staging area, the dot after **add** in the first command tells Git to check for changes in all the files. The second step is about committing your changes. The message part is short description of your commit, what changes that particular commit is doing. For example, if while creating a login form, the commit message can be something like `creating login form`.
+
+```
+git add .
+git commit -m "Write message about your commit"
+```
+
+8. **Making sure you have the latest changes from the develop branch**
+
+It may so happen that since the last time you cloned the repo/took a pull from develop, some changes may be merged in the develop branch. So to be on the safer side, we should have those changes as well.
+
+In order to do that, we first checkout to **develop** branch by:
+
+```
+git checkout develop
+```
+
+Once we're in develop, it's time to take a pull:
+
+```
+git pull upstream develop
+```
+
+Now that our **local** develop branch is in sync with **remote** develop branch (of the Real Dev Squad Repository), we should let our branch know about the changes from the develop branch (if any). To do so we first checkout to our branch:
+
+```
+git checkout <branch-name>
+```
+
+Once we're in our branch, we **rebase** our branch on top of the current develop branch (we change the base of our branch, so that it appears as if we have worked from the time the latest changes were merged in the develop branch). To do so:
+
+```
+git rebase develop
+```
+
+You should solve the merge conflicts, if any.
+
+9. **Pushing your code**
+
+Now it is the time to push the changes of your local develop branch to your fork of the repository which is on Github (which we cloned in step 2).
+
+```
+git push origin <branch-name>
+```
+
+10. **Pull request**
+
+Now go to your forked repository and you'll see that **This branch is xx commits ahead of Real-Dev-Squad:develop.** Right next to it will be an option to **Pull Request**. Click on it, submit your pull request (also known as _PR_) explaining what you've done.
+
+![how-to-open-pull-request](https://i.imgur.com/NMAeWc2.png)
+
+11. **Review stage**
+
+Wait for it to get reviewed, make the changes required (if any) , commit your changes and hit **Pull Request** again, your commits will be added to the same pull request you had opened earlier (if it is NOT closed).
```

Comment:
Here, instead of commit and pull request, you can add steps of commit and push or just mention which steps to repeat.